lib/mountain_view/component.rb

```diff
@@ -14,10 +14,18 @@ def styleguide_stubs
       YAML.load_file stubs_file
     end
 
-    private
-
     def stubs_file
       Rails.root.join("app/components/#{name}/#{name}.yml")
     end
+
+    def stubs?
+      if styleguide_stubs.is_a?(Array)
+        styleguide_stubs.any?
+      else
+        false
+      end
```

Comment:
What do you think about doing a one-liner kind of thing? The `else false` part seems a bit redundant, both `is_a?` and `any?` already return the `true`/`false` we want.

Something like the following could work.
```
styleguide_stubs.is_a?(Array) && styleguides_stubs.any?
```